.env.development

```
@@ -0,0 +1,10 @@
+# Google Sheets API Configuration
+GOOGLE_PROJECT_ID=eliksir-api
+GOOGLE_PRIVATE_KEY_ID=4e86f2d17625796516d8e1a568935d3bd30fdf9d
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GOOGLE_PRIVATE_KEY_ID key should go before the GOOGLE_PROJECT_ID key

(UnorderedKey)